lib/src/shops/shop_details_screen.dart

```diff
@@ -43,7 +43,7 @@ class ShopDetailScreen extends StatelessWidget {
                   child: ListView(
                     children: [
                       Image.network(
-                          "${BedrockService.api}/${BedrockService.apiVersion}/uploads/${shop.images[0].id}/image"),
+                          "${BedrockService.baseURL}/${BedrockService.apiVersion}/uploads/${shop.images[0].id}/image"),
```

Comment:
We should probably wrap this in a utility function or component (uploads are standard objects)